src/ngraph/runtime/plaidml/plaidml_ops_general.cpp

```diff
@@ -209,10 +209,16 @@ void ngraph::runtime::plaidml::ImplPad::Apply()
 
     NGRAPH_DEBUG << "Pad below: " << op().get_padding_below();
     NGRAPH_DEBUG << "Pad above: " << op().get_padding_above();
-    NGRAPH_DEBUG << "Pad interior: " << op().get_padding_interior();
     NGRAPH_DEBUG << "Pad input dims: " << op().get_input_shape(0);
     NGRAPH_DEBUG << "Pad output dims: " << op().get_shape();
 
+    // FIXME: Compatibility hack inserted by amprocte, now that nGraph's Pad op no longer supports
+    // interior padding.
+    Shape padding_interior(op.get_padding_below().size(), 0);
+
+    // FIXME: Check inserted by amprocte, should support the new modes as well.
+    NGRAPH_ASSERT(op.get_pad_mode() == op::PadMode::CONSTANT);
```

Comment:
Please add tests for this (and other new functionality related to the operation) and if it failed - add them into unit_test.manifest. I will fix them later.

Reply:
Some tests are added by @aprocter in `test/backend_test.in.cpp` is it enough?

Reply:
If you think these tests are enough to cover functionality it is ok.